Packages/Tracking/OpenXR/Runtime/Scripts/OpenXRLeapProvider.cs

```diff
@@ -33,6 +33,18 @@ public class OpenXRLeapProvider : LeapProvider
 
         // Magic numbers for palm width and PinchStrength calculation
         private static readonly float[] DefaultMetacarpalLengths = { 0, 0.06812f, 0.06460f, 0.05800f, 0.05369f };
+        private static readonly Quaternion[] ThumbMetacarpalRotationOffset =
+        {
+            Quaternion.Euler(0, +HAND_ROTATION_OFFSET_Y, +HAND_ROTATION_OFFSET_Z),
```

Comment:
I'd love to make these constants (`HAND_ROTATION_OFFSET_Y` and `HAND_ROTATION_OFFSET_Z`) non-public, but I think that ship has sailed until a major version bump.

Reply:
I agree, though it would require that bump due to being public :( 

We could make new private ones, and then mark the public as obsolete and unused?